libr/anal/p/anal_alpha.c

```diff
@@ -61,26 +62,37 @@ static int disassemble(RAsm *a, RAsmOp *op, const ut8 *buf, int len) {
 	disasm_obj.stream = stdout;
 	op->size = print_insn_alpha ((bfd_vma)Offset, &disasm_obj);
 
-	if (op->size == -1) {
-		r_asm_op_set_asm (op, "(data)");
+	if (mask & R_ANAL_OP_MASK_DISASM) {
+		if (op->size > 0) {
+			op->mnemonic = r_strbuf_drain (buf_global);
+			for (char *c = op->mnemonic; *c != 0; c++) {
```

Comment:
use `r_str_replace_char (op->mnemonic, '\t', ' ', 0);`
```suggestion
			r_str_replace_char (op->mnemonic, '\t', ' ', 0);
```